pyoozie/client.py

```diff
@@ -561,6 +561,25 @@ def jobs_submit_coordinator(self, xml_path, configuration=None):
             return coord
         raise exceptions.OozieException.operation_failed('submit coordinator')
 
+    def jobs_update_coordinator(self, xml_path, coordinator_id, configuration=None):
+        user = self._user or 'oozie'
+        coord = self._fetch_coordinator_or_action(coordinator_id)
+        if coord.status.is_active():
+            conf = xml._coordinator_submission_xml(user, xml_path, configuration=configuration)
+            if self._verbose:
+                self.logger.info('Preparing to update coordinator %s:\n%s', xml_path, conf)
+            reply = self._put('job/{}?action=update'.format(coordinator_id), conf)
+
+            if 'update' not in reply:
+                raise exceptions.OozieException.operation_failed('update coordinator')
+
+            if self._verbose:
+                self.logger.info('Coordinator %s updated with diff %s', coordinator_id, reply['update']['diff'])
+
+            return self.job_coordinator_info(coordinator_id=coordinator_id)
```

Comment:
I agree, my idea is that you could extract the "diff" by looking at the coordinator result and comparing it to the input coordinator. Returning the diff result string doesn't seem to fit the rest of the API.